documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -536,7 +536,7 @@ public void renameDocument(long ownerId, String documentID, String title, Identi
       String oldName = node.getName();
       if (oldName.indexOf('.') != -1 && node.isNodeType(NodeTypeConstants.NT_FILE)) {
         String ext = oldName.substring(oldName.lastIndexOf('.'));
-        title = title.concat(ext);
+        title = name.concat(ext);
```

Comment:
No need to change the title, the title is a property that does not need to be cleaned nor escaped its illegal JCR characters

Reply:
Also I don't get why do we need to do cleanString twice in lines 530 and 532